lib/cocoapods-search/command/search.rb

```diff
@@ -82,7 +84,19 @@ def local_search
           sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
         end
 
-        sets.reverse_each do |set|
+        if use_pager
+          extend_ui_puts
+          IO.popen('$PAGER', 'w') do |io|
```

Comment:
Or maybe we can add a `--no-pager` option like git which does not use a
pager?
On Sun, 8 Nov 2015 at 20:30, Kyle Fuller <notifications@github.com> wrote:

> In lib/cocoapods-search/command/search.rb
> <https://github.com/CocoaPods/cocoapods-search/pull/18#discussion_r44228179>
> :
>
> > @@ -82,7 +84,19 @@ def local_search
> >            sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
> >          end
> >
> > -        sets.reverse_each do |set|
> > +        if use_pager
> > +          extend_ui_puts
> > +          IO.popen('$PAGER', 'w') do |io|
>
> I think we should resolve this ourselves and fallback to less/more in the
> case where a user doesn't have an explicit PAGER set?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CocoaPods/cocoapods-search/pull/18/files#r44228179>.
>